### PR TITLE
Add festival picker to fest show

### PR DESCRIPTION
**Pull request by lancekrogers**
## Summary

- Opens the shared festival picker when `fest show` is run outside a festival but inside an interactive campaign workspace.
- Keeps JSON and non-interactive behavior deterministic, with an actionable `--festival <selector>` hint when no picker is available.
- Reuses the existing picker candidate ordering for active, ready, planning, and ritual festivals.

## Validation

- `go test ./internal/commands/show`
- `just test unit`
- `just lint vet`
- `just lint all`
- `just build quick`

### CODE REVIEW

File: internal/commands/show/show.go [hunk 1]
```diff
@@ -229,6 +246,21 @@ func runShowCurrent(ctx context.Context, opts *showOptions) error {
 	return emitShowFestival(ctx, festival, opts, campaignRoot)
 }
 
+var showPickerStatuses = []string{"active", "ready", "planning", "ritual"}
```
Comment (obey-agent) [inline internal/commands/show/show.go]: **Blocker — cross-command consistency:** `showPickerStatuses` includes `ritual`, but `fest status` (`statusPickerStatuses`) and `fest progress` (`progressPickerStatuses`) only use `active`, `ready`, `planning`. Running `fest show` vs `fest status` from the same project directory will surface different picker candidate pools. If ritual is intentional for show, document why; otherwise align the shared status slice across commands.

File: internal/commands/show/show_command_test.go [hunk 2]
```diff
@@ -79,6 +82,126 @@ func TestRunShowBySelector_RequiresCampaignWorkspace(t *testing.T) {
 	}
 }
 
+func TestRunShowCurrent_PicksFestivalOutsideFestival(t *testing.T) {
```
Comment (obey-agent) [inline internal/commands/show/show_command_test.go:85]: **Blocker — test parity:** `internal/commands/status/picker_fallback_test.go` already covers no-campaign, no-`.festival` marker, and empty-festivals picker behavior. This PR adds `pickFestivalForShow` but does not include equivalent direct tests — only mocked `showPickFestival` integration tests. Please add `pickFestivalForShow` fallback tests matching the status package so regressions in FindFestivals/unavailable handling are caught without mocking the picker.

**Review by obey-agent — CHANGES_REQUESTED** [review]
## Staff Review — Request Changes

**Verdict:** Request Changes

I missed this PR in the initial org pass (it opened after that search). Re-reviewed adversarially with tests run on the PR branch.

**What's good:**
- Reuses `shared.PickFestival` with proper outcome handling (picked / cancelled / unavailable)
- JSON mode stays deterministic and does not invoke the picker (tested)
- Non-interactive fallback includes `--festival <selector>` hint (tested)
- `go test ./internal/commands/show` passes on `show-festival-picker`

**Blockers:**
1. **Picker status set inconsistency** — show includes `ritual`; `fest status` and `fest progress` pickers do not. Same cwd, different candidate pools (inline comment).
2. **Missing picker_fallback parity tests** — status already has direct `pickFestivalForDisplay` edge-case tests; show only mocks `showPickFestival` (inline comment).

**Staff standard:** Close, but not merge-ready until cross-command picker consistency and fallback test parity are resolved.

**Review by obey-agent — APPROVED** [review]
## Staff Review — Approved

**Verdict:** Approve

Follow-up commit `6302182` resolves both prior blockers:

1. **Picker status consistency** — `WorkingFestivalPickerStatuses` (status/progress/watch) and `BrowseFestivalPickerStatuses` (show, includes ritual) are now centralized in `shared` with documented intent.
2. **Test parity** — `internal/commands/show/picker_fallback_test.go` covers no-campaign, no-marker, and empty-festivals paths for `pickFestivalForShow`.

Re-verified: `go test ./internal/commands/show ./internal/commands/shared ./internal/commands/status ./internal/commands/progress` passes on the updated branch.

**Review by obey-agent — APPROVED** [review]
## Staff Review (re-check) — Approve

**Verdict:** Approve

No new commits since prior approval at `6302182`. Blockers remain resolved: shared `WorkingFestivalPickerStatuses` / `BrowseFestivalPickerStatuses`, plus `picker_fallback_test.go` for show.

**Staff standard:** Yes.
